src/components/ui/UiTooltip.vue

```diff
@@ -0,0 +1,38 @@
+<script lang="ts" setup>
```

Comment:
Maybe rename the file with a more explicit name ? 
Like UiExplanationTooltip ? 
Because we have a lot of tooltip in the project: UiButtonWithTooltip from common, DistrictDataTooltip,AddressTooltip, ... 
![image](https://user-images.githubusercontent.com/121813008/229947901-f7e28c62-ba24-42fb-ae5c-9f773d226ce6.png)